release_description.md

```diff
@@ -0,0 +1,14 @@
+### The Amazon QLDB team is pleased to announce the release of v1.2.0 of the QLDB .NET driver. This release adds asynchronous support to the driver.
```

Comment:
This shouldn't be a title format, remove ###

Reply:
Done.